crates/scheduler/src/call/store.rs

```diff
@@ -57,16 +57,16 @@ fn insert<B: Board>(mut call: SchedulerCall<B, api::insert::Sig>) {
     let memory = scheduler.applet.memory();
     let result = try {
         let value = memory.get(*ptr, *len)?;
-        scheduler.store.insert(*key as usize, value).map_err(convert)
+        scheduler.store.insert(*key as usize, value).map_err(convert)?
     };
     call.reply(result);
 }
 
 #[cfg(feature = "board-api-storage")]
 fn remove<B: Board>(mut call: SchedulerCall<B, api::remove::Sig>) {
     let api::remove::Params { key } = call.read();
-    let res = call.scheduler().store.remove(*key as usize).map_err(convert);
-    call.reply(Ok(res));
+    let res = call.scheduler().store.remove(*key as usize).map_err(|x| convert(x).into());
```

Comment:
nit: a try-block would help here
```suggestion
    let res = try { call.scheduler().store.remove(*key as usize).map_err(convert)? };
```